### PR TITLE
[Sofia]Fetch task

### PR DESCRIPTION
**Pull request by SofiMat**

### CODE REVIEW

File: html-template-3/export.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+function fetchData() {
+  return fetch('http://localhost:3000/api/blogs')
+      .then((response) => {
+        return response.json();
+      })
+      .then(dataFilter);
+}
+
+export const formatDate = (date) => {
```
Comment (dverbovyi) [inline html-template-3/export.js]: move this function to blog.js because it's not using here

File: html-template-3/export.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+/* global fetch */
```
Comment (dverbovyi) [inline html-template-3/export.js]: give a more appropriate name for this file

File: html-template-3/export.js (hunk 1 above)
Comment (SofiMat) [reply html-template-3/export.js]: done

File: html-template-3/export.js (hunk 2 above)
Comment (SofiMat) [reply html-template-3/export.js]: done

**Pull request merged**